### ISSUE TITLE
Can contribute to display Chinese characters with a customed font(bitmap) on Non-APAC devices?

### ISSUE THREAD

**Issue by Likenttt** [warmsound/crystal-face#202]
Non-APAC devices cannot display CJK Unicode without a customed font. For example, my 245m is the German version, the system language is limited to some European languages, in which case any CJK Unicode will display as a  garbled character. However, I want to display the lunar date in Chinese characters that is unavailable right now. In my opinion, there are two solutions:
1. Add a setting item in resources-eng, they can choose to display with Chinese characters or not. If the item is chosen, use the customed font to draw CJK text. But I think it is not necessary for most English users. And it will import complexity, they may wonder what is the lunar date. So another solution is :
2. Fork this project and remove all language supports excluding English and Simplified Chinese. Add the customed font in resources-eng.

In conclusion, I want to add support for the lunar date in APAC and add customed CJK font support for non-APAC devices.
Have a good day!
